projects/ngx-piwik-pro/package.json

```diff
@@ -24,6 +24,7 @@
     "@angular/core": ">=13.2.0"
   },
   "dependencies": {
-    "tslib": "^2.6.2"
+    "tslib": "^2.6.2",
```

Comment:
Question: Could `tslib` be peer dependency? Doesn't every angular app depend on `tslib`? This means that every project that uses our package will have two versions of `tslib` Can't we relay on project one?

Reply:
done